packages/@coorpacademy-components/src/molecule/learner-skill-card/style.css

```diff
@@ -0,0 +1,116 @@
+@value breakpoints: "../../variables/breakpoints.css";
+@value mobile from breakpoints;
+@value colors: "../../variables/colors.css";
+@value cm_grey_400 from colors;
+@value white from colors;
+@value light from colors;
+@value box_shadow_light_dark from colors;
+@value box_shadow_medium_dark from colors;
+
+.textBase {
+  font-family: "Gilroy";
+  font-style: normal;
+}
+
+.learnerSkillCardWrapper {
+  display: flex;
+  flex-direction: column;
+  max-width: 258px;
+  width: 100%;
+  min-height: 252px;
+  box-shadow: 0 4px 16px 0px rgba(0, 0, 0, 0.4);;
+  transition: box-shadow 0.15s ease-in-out;
+  border-radius: 12px;
+  padding: 24px;
+  box-sizing: border-box;
+}
+
+.learnerSkillCardWrapper:hover {
+  box-shadow: 0 4px 40px box_shadow_medium_dark;
+}
+
+.skillCoursesAndQuestionsWrapper,
+.progressInformations {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 8px;
+}
+
+.skillInformation,
+.progressInformation {
+  composes: textBase;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 12px;
+  color: cm_grey_400;
+}
+
+.skillInformationNumber,
+.progressInformationNumber {
+  color: #000;
+}
+
+.skillTitleWrapper {
+  display: flex;
+  flex-direction: column;
+  min-height: 82px;
+  margin-bottom: 24px;
+}
+
+.skillTitle {
+  composes: textBase;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  margin-bottom: 8px;
+}
+
+.skillFocusBadge {
+  composes: textBase;
+  display: flex;
+  height: 14px;
+  width: 62px;
+  align-items: center;
+  background-color: #DDD1FF;
+  border-radius: 24px;
+  padding: 2px 4px;
+  font-size: 10px;
+  font-weight: 400;
+  color: #3900D6;
+  margin-right: 8px;
+}
+
+.progressWrapper {
+  position: relative;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 2px;
+  overflow: hidden;
+  background: light;
+  margin-bottom: 8px;
+}
+
+.progress {
+  display: block;
+  height: 4px;
+  animation: progressbar 2s ease-in-out;
```

Comment:
progressbar is a reference at?

Reply:
Ah, nice catch! I'll update it. thank you Adam